apps/src/templates/levelSummary/AiFeedbackType.tsx

```diff
@@ -0,0 +1,30 @@
+import color from '@cdo/apps/util/color';
+import i18n from '@cdo/locale';
+
+const makeObjectType = (title: string, icon: string, color: string) => ({
```

Comment:
TBH, I am not 100% sure if this file is neccessary... I thought I would give it a try for now - we might decide it isn't worth the hassle and refactor later...

Reply:
🤷‍♀️ I don't feel strongly either way. I think it's partly about what you feel like would be easier to reason about and maintain over time